manifests/controller-manager/vsphere-cloud-controller-manager-ds.yaml

```diff
@@ -26,7 +30,7 @@ spec:
       nodeSelector:
         node-role.kubernetes.io/master: ""
       securityContext:
-        runAsUser: 0
```

Comment:
Are the changes in this file intentional?

Reply:
yes, it is. I forgot to pull down the YAML I used to test with. There were discrepancies between the vsphere-cloud-controller-manager-ds.yaml and vsphere-cloud-controller-manager-pod.yaml. This resolves it... the ds.yaml should be running as user 1001 which is a non-root account.

Reply:
Should have been changed for this PR:
https://github.com/kubernetes/cloud-provider-vsphere/pull/297